src/components/Book.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components'
+
+function Book(props) {
+  return (
+    <Wrapper>
+      <Img>
+        <div>
+          <img src={props.bookCover} alt="bookcover" />
+        </div>
+      </Img>
+      <Content>
+        <div>
+          <h3>{props.title}</h3>
+          <p>
+            by
+            {props.authors.length > 1
+              ? ` ${props.authors[0]}`
+              : ` ${props.authors}`}
+          </p>
+        </div>
+        <div>
+          <div>{!props.readingStatus ? '' : 'Rating'}</div>
+          <div>
+            {!props.readingStatus
+              ? `Reading since: ${props.readingStatusDate}`
+              : `
+        Finished since: ${props.finishedSince}`}
```

Comment:
Something went wrong with your "Einrückung". Please check again if this is correct 🙂